bin/env_data_store.txt

```diff
@@ -0,0 +1,35 @@
+TEST_URL=http://localhost:4452
```

Comment:
please order the lines alphabetically for easier lookup and editing later.

Reply:
Reordered the env variables. However, I have kept some variables in pairs so that readability can be maintained. For example, we have the following email and password pair.

CCD_RESTRICTED_CASEWORKER_EMAIL=auto.test.cnp+senior@gmail.com
CCD_RESTRICTED_CASEWORKER_PASSWORD=password